include/reshade_api.hpp

```diff
@@ -660,5 +667,11 @@ namespace reshade::api
 			size_t length = SIZE;
 			get_technique_effect_name(technique, effect_name, &length);
 		}
+
+		/// <summary>
+		/// *TODO*
+		/// </summary>
+		/// <returns>*TODO*</returns>
+		virtual bool has_effects_loaded() const = 0;
```

Comment:
Why is this needed? Events are never called during effect loading anyway, so this should not be needed. You can use `reshade_reloaded_effects` to keep track of when effects are loaded.